db/agents.yml

```diff
@@ -262,47 +310,59 @@ agents:
 
   # witness ye podcasts and despair
   - regex: ^Podcasts
-    name: Podcasts
+    name: Apple Podcasts
     type: Mobile App
     os: iOS
   - regex: ^itunesstored
-    name: Podcasts
+    name: Apple Podcasts
     type: Mobile App
     os: iOS
   - regex: ^AppleCoreMedia.+Apple TV
-    name: Podcasts
+    name: Apple Podcasts
     type: Smart TV
     os: iOS
   - regex: ^AppleCoreMedia.+(Audio Accessory|HomePod)
-    name: Podcasts
+    name: Apple Podcasts
     type: Smart Home
     os: iOS
   - regex: ^AppleCoreMedia.+iPad
-    name: Podcasts
+    name: Apple Podcasts
     type: Mobile App
     os: iOS
   - regex: ^AppleCoreMedia.+iPhone
-    name: Podcasts
+    name: Apple Podcasts
     type: Mobile App
     os: iOS
   - regex: ^AppleCoreMedia.+iPod
-    name: Podcasts
+    name: Apple Podcasts
     type: Mobile App
     os: iOS
+  - regex: ^AppleCoreMedia.+(Apple Watch)
+    name: Apple Podcasts
+    type: Smart Watch
+    os: WatchOS
+  - regex: \bwatchOS/
+    name: Apple Podcasts
+    type: Smart Watch
+    os: WatchOS
   - regex: ^AppleCoreMedia.+Macintosh
     name: iTunes
     type: Desktop App
     os: macOS
 
   # google search (including "google podcasts" app)
-  - regex: ^Mozilla/5\.0.+Android.+GSA
-    name: Google Search App
+  - regex: ^Mozilla/5\.0.+Android.+GSA/
+    name: Google Podcasts
     type: Mobile App
     os: Android
-  - regex: ^Mozilla/5\.0.+iPhone.+GSA
-    name: Google Search App
+  - regex: ^Mozilla/5\.0.+iPhone.+GSA/
+    name: Google Podcasts
     type: Mobile App
     os: iOS
+  - regex: \bGoogleChirp\/
+    name: Google Podcasts
+    type: Smart Home
+    os: Google Home
```

Comment:
Or should this just be "Android"?  I also have a "Sonos OS" and "Amazon OS".  But Roku has null for `os`, and HomePod has "iOS".  So I guess I haven't been entirely consistent.  Suggestions?

Reply:
Also have "WatchOS" for apple watch, which _is_ actually an OS, i guess?